debian-template/wiringPi/DEBIAN/control

```diff
@@ -1,10 +1,13 @@
 Package: wiringpi
-Version: 2.61
+Version: 2.70
 Section: libraries
 Priority: optional
 Architecture: armhf
 Depends: libc6
-Maintainer: Gordon Henderson <projects@drogon.net>
+Maintainer: Mark Liffiton <liffiton@gmail.com>
```

Comment:
@liffiton I suspect you'd rather *not* be included as maintainer here, since this is more of a light-touch mirror and not a fully maintained fork?

Reply:
Yes, that's correct.  I am not involved enough to be the maintainer of the Debian package.  Thank you for checking.

Reply:
In that case, I'll stick myself in there for the time being with the hope that someone else will step up. :grimacing:

Reply:
That's generous of you.  Fwiw, I'll still be able to help a bit here and there, but usually with a delay before I get a chunk of time for it.

Reply:
Thanks for all of this - it looks like progress :)  Curious about your stmt: 

>"... since its method of bit-banging GPIO registers directly has a very tenuous future with modern, upstream-compatible Linux favoring libgpiod. 

Where could I learn more about this issue? Are you referring to the *chardev* GPIO interface?

Reply:
Yes, `chardev` is the interface, `libgpiod` is a library for interfacing with it. I tend to refer to it by the latter since it was my first encounter with chardev GPIO.